src/components/CheckListContainer.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import CheckList from "./CheckList";
+import { Container, Row } from "react-bootstrap";
+import "./CheckList.css";
+
+const ChecklistContainer = ({ data }) => {
+  console.log(data);
+  return (
+    <Container>
+      <Row className="py-5">
+        {data.map((info, index) => (
+          <CheckList
+            text={info.text}
+            title={info.title}
+            link={info.link}
+            key={index}
```

Comment:
lets not use indexes as keys please resources: https://dev.to/rajnishkatharotiya/you-shouldn-t-use-an-index-as-key-of-the-element-2jf